common/frame.go

```diff
@@ -1146,27 +1146,41 @@ func (f *Frame) innerText(selector string, opts *FrameInnerTextOptions) (string,
 	return gv.ToString().String(), nil
 }
 
+// InputValue returns the input value of the first element found
+// that matches the selector.
 func (f *Frame) InputValue(selector string, opts goja.Value) string {
 	f.log.Debugf("Frame:InputValue", "fid:%s furl:%q sel:%q", f.ID(), f.URL(), selector)
 
-	parsedOpts := NewFrameInputValueOptions(f.defaultTimeout())
-	if err := parsedOpts.Parse(f.ctx, opts); err != nil {
-		k6ext.Panic(f.ctx, "%w", err)
+	popts := NewFrameInputValueOptions(f.defaultTimeout())
+	if err := popts.Parse(f.ctx, opts); err != nil {
+		k6ext.Panic(f.ctx, "parse: %w", err)
+	}
+	v, err := f.inputValue(selector, popts)
+	if err != nil {
+		k6ext.Panic(f.ctx, "InputValue: %w", err)
```

Comment:
This should be camelCase, not PascalCase, as that's how it's exposed in JS.

```suggestion
		k6ext.Panic(f.ctx, "inputValue: %w", err)
```